diff --git a/common/lcd.c b/common/lcd.c
index d854c21..b9832a1 100644
--- a/common/lcd.c
+++ b/common/lcd.c
@@ -56,8 +56,11 @@
 /************************************************************************/
 /* ** FONT DATA								*/
 /************************************************************************/
+#ifdef	CONFIG_CONSOLE_ROTATE_270
+#include <video_font_8x8_rot270.h>	/* Get font data, width and height	*/
+#else
 #include <video_font.h>		/* Get font data, width and height	*/
-
+#endif
 /************************************************************************/
 /* ** LOGO DATA								*/
 /************************************************************************/
@@ -96,7 +99,7 @@ static int lcd_getfgcolor (void);
 /************************************************************************/
 
 /*----------------------------------------------------------------------*/
-
+#ifndef	CONFIG_CONSOLE_ROTATE
 static void console_scrollup (void)
 {
 	/* Copy up rows ignoring the first one */
@@ -175,7 +178,106 @@ void lcd_putc (const char c)
 	}
 	/* NOTREACHED */
 }
+#endif
+#ifdef	CONFIG_CONSOLE_ROTATE_270
+/*
+ Rotate the console output 270 degrees (=90 degrees CCW)
+ Idea: let's keep everything related to framebuffer dimensions as is (colums, rows, font width, heigth) 
+ but redefine the way of writing. We are writing bottom up from bottom left column (x=0,y=CONSOLE_ROWS-1)
+ to upper right column (x=CONSOLE_COLS-1,y=0), each 'line' is now one column so :
+ - newline means move to last row and increase column
+ - next character is one row up
+ - scrolling 'line' means scrolling one column to left and keep writing in last column
+ - we also need pre-rotated font to make all this look normal
+*/
+static void console_scroll_rot270(void)
+{
+	/* Clear the first character in each row (will be moved to last column when scrolling) */
+	{
+		char *p=lcd_console_address;
+		char *end=p+CONSOLE_SIZE;
+		while (p<end) {
+			memset (p, COLOR_MASK(lcd_color_bg), VIDEO_FONT_WIDTH);
+			p+=lcd_line_length;
+		}
+	}
+	/* Now copy columns to left by pulling all pixel lines to left by character width.
+	First column will be automatically moved to last column and scrolled by one pixel line up
+	but since we clear it in advance it is blank so the one line up does not matter */
+	memcpy (lcd_console_address, lcd_console_address+VIDEO_FONT_WIDTH, CONSOLE_SIZE-VIDEO_FONT_WIDTH);
+
+	/* Now also clear last pixel line of last character in bottom right corner. */
+	memset (lcd_console_address+CONSOLE_SIZE-VIDEO_FONT_WIDTH, COLOR_MASK(lcd_color_bg), VIDEO_FONT_WIDTH);
+
+}
+
+static inline void console_newline_rot270 (void)
+{
+	console_row = CONSOLE_ROWS-1; // CR
+
+	/* Check if we need to scroll the terminal */
+	if (console_col < CONSOLE_COLS-1)
+		console_col++; // LF
+	else {
+		/* Scroll everything left */
+		console_scroll_rot270() ;
+	}
+}
+static inline void console_back_rot270 (void)
+{
+	console_row++;
+	if (console_row >= CONSOLE_ROWS) {
+		console_row = 0 ;
+		if (console_col > 0) {
+			console_col--;
+		}
+	}
+
+	lcd_putc_xy (console_col * VIDEO_FONT_WIDTH,
+		     console_row * VIDEO_FONT_HEIGHT,
+		     ' ');
+}
+void lcd_putc_rot270 (const char c)
+{
+	if (!lcd_is_enabled) {
+		serial_putc(c);
+		return;
+	}
+
+	switch (c) {
+	case '\r':	console_row = CONSOLE_ROWS-1;
+			return;
+
+	case '\n':	console_newline_rot270();
+			return;
+
+	case '\t':	/* Tab (8 chars alignment) */
+			if (console_row < 8) {
+				console_newline_rot270();
+			} else {
+				console_row -=  7;
+				console_row &= ~7;
+				console_row -= 1;
+			}
+			return;
+
+	case '\b':	console_back_rot270();
+			return;
+
+	default:	lcd_putc_xy (console_col * VIDEO_FONT_WIDTH,
+				     console_row * VIDEO_FONT_HEIGHT,
+				     c);
+			if (console_row == 0) {
+				console_newline_rot270();
+			} else
+				console_row--;
+			return;
+	}
+	/* NOTREACHED */
+}
 
+#define	lcd_putc	lcd_putc_rot270
+#endif
 /*----------------------------------------------------------------------*/
 
 void lcd_puts (const char *s)
@@ -261,8 +363,10 @@ static void lcd_drawchars (ushort x, ushort y, uchar *str, int count)
 	}
 }
 
+#ifndef CONFIG_CONSOLE_ROTATE
 /*----------------------------------------------------------------------*/
-
+// not used anyway, we must check end of line after each character and
+// also handle control characters (cr,lf,tab,...)
 static inline void lcd_puts_xy (ushort x, ushort y, uchar *s)
 {
 #if defined(CONFIG_LCD_LOGO) && !defined(CONFIG_LCD_INFO_BELOW_LOGO)
@@ -271,8 +375,8 @@ static inline void lcd_puts_xy (ushort x, ushort y, uchar *s)
 	lcd_drawchars (x, y, s, strlen ((char *)s));
 #endif
 }
-
 /*----------------------------------------------------------------------*/
+#endif
 
 static inline void lcd_putc_xy (ushort x, ushort y, uchar c)
 {
@@ -388,10 +492,14 @@ static int lcd_clear (cmd_tbl_t * cmdtp, int flag, int argc, char * const argv[]
 	/* Paint the logo and retrieve LCD base address */
 	debug ("[LCD] Drawing the logo...\n");
 	lcd_console_address = lcd_logo ();
-
+#ifndef	CONFIG_CONSOLE_ROTATE
 	console_col = 0;
 	console_row = 0;
-
+#endif
+#ifdef	CONFIG_CONSOLE_ROTATE_270
+	console_col = 0;
+	console_row = CONSOLE_ROWS-1;
+#endif
 	return (0);
 }
 
@@ -413,14 +521,15 @@ static int lcd_init (void *lcdbase)
 	lcd_clear (NULL, 1, 1, NULL);	/* dummy args */
 	lcd_enable ();
 
-	/* Initialize the console */
-	console_col = 0;
+	/* Initialize the console, done also in lcd_clear call above */
+////	console_col = 0;
+#ifndef	CONFIG_CONSOLE_ROTATE
 #ifdef CONFIG_LCD_INFO_BELOW_LOGO
 	console_row = 7 + BMP_LOGO_HEIGHT / VIDEO_FONT_HEIGHT;
 #else
 	console_row = 1;	/* leave 1 blank line below logo */
 #endif
-
+#endif
 	return 0;
 }
 
diff --git a/include/configs/zipitz2.h b/include/configs/zipitz2.h
index cdfcdcc..83ef090 100644
--- a/include/configs/zipitz2.h
+++ b/include/configs/zipitz2.h
@@ -98,6 +98,8 @@
 #define	CONFIG_CMD_BMP
 #define	CONFIG_SPLASH_SCREEN
 #define	CONFIG_SPLASH_SCREEN_ALIGN
+#define CONFIG_CONSOLE_ROTATE
+#define CONFIG_CONSOLE_ROTATE_270
 #define	CONFIG_VIDEO_BMP_GZIP
 #define	CONFIG_VIDEO_BMP_RLE8
 #define	CONFIG_SYS_VIDEO_LOGO_MAX_SIZE	(2 << 20)
diff --git a/include/lcd.h b/include/lcd.h
index cd9d49d..ec12ba1 100644
--- a/include/lcd.h
+++ b/include/lcd.h
@@ -312,12 +312,15 @@ void lcd_show_board_info(void);
 
 #define CONSOLE_COLS		(panel_info.vl_col / VIDEO_FONT_WIDTH)
 #define CONSOLE_ROW_SIZE	(VIDEO_FONT_HEIGHT * lcd_line_length)
+#define CONSOLE_SIZE		(CONSOLE_ROW_SIZE * CONSOLE_ROWS)
+
+#ifndef	CONFIG_CONSOLE_ROTATE
 #define CONSOLE_ROW_FIRST	(lcd_console_address)
 #define CONSOLE_ROW_SECOND	(lcd_console_address + CONSOLE_ROW_SIZE)
 #define CONSOLE_ROW_LAST	(lcd_console_address + CONSOLE_SIZE \
 					- CONSOLE_ROW_SIZE)
-#define CONSOLE_SIZE		(CONSOLE_ROW_SIZE * CONSOLE_ROWS)
 #define CONSOLE_SCROLL_SIZE	(CONSOLE_SIZE - CONSOLE_ROW_SIZE)
+#endif
 
 #if LCD_BPP == LCD_MONOCHROME
 # define COLOR_MASK(c)		((c)	  | (c) << 1 | (c) << 2 | (c) << 3 | \
diff --git a/include/video_font_8x8_rot270.h b/include/video_font_8x8_rot270.h
new file mode 100644
index 0000000..caf6834
--- /dev/null
+++ b/include/video_font_8x8_rot270.h
@@ -0,0 +1,2316 @@
+#ifndef _VIDEO_FONT_
+#define _VIDEO_FONT_
+
+
+#define VIDEO_FONT_CHARS        256
+#define VIDEO_FONT_WIDTH        8
+#define VIDEO_FONT_HEIGHT       8
+#define VIDEO_FONT_SIZE         (VIDEO_FONT_CHARS * VIDEO_FONT_HEIGHT)
+
+static unsigned char video_fontdata[VIDEO_FONT_SIZE] = {
+	/* 0 = 0x00 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 1 = 0x01 90*/
+	0x7c, /*   .XXXXX..   */
+	0x82, /*   X.....X.   */
+	0xaa, /*   X.X.X.X.   */
+	0x8a, /*   X...X.X.   */
+	0x8a, /*   X...X.X.   */
+	0xaa, /*   X.X.X.X.   */
+	0x82, /*   X.....X.   */
+	0x7c, /*   .XXXXX..   */
+	/* 2 = 0x02 90*/
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0xd6, /*   XX.X.XX.   */
+	0xf6, /*   XXXX.XX.   */
+	0xf6, /*   XXXX.XX.   */
+	0xd6, /*   XX.X.XX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	/* 3 = 0x03 90*/
+	0x70, /*   .XXX....   */
+	0xf8, /*   XXXXX...   */
+	0xfc, /*   XXXXXX..   */
+	0x7e, /*   .XXXXXX.   */
+	0xfc, /*   XXXXXX..   */
+	0xf8, /*   XXXXX...   */
+	0x70, /*   .XXX....   */
+	0x00, /*   ........   */
+	/* 4 = 0x04 90*/
+	0x10, /*   ...X....   */
+	0x38, /*   ..XXX...   */
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x38, /*   ..XXX...   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 5 = 0x05 90*/
+	0x1c, /*   ...XXX..   */
+	0x1c, /*   ...XXX..   */
+	0xc9, /*   XX..X..X   */
+	0xff, /*   XXXXXXXX   */
+	0xc9, /*   XX..X..X   */
+	0x1c, /*   ...XXX..   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 6 = 0x06 90*/
+	0x10, /*   ...X....   */
+	0x39, /*   ..XXX..X   */
+	0x7b, /*   .XXXX.XX   */
+	0xff, /*   XXXXXXXX   */
+	0x7b, /*   .XXXX.XX   */
+	0x39, /*   ..XXX..X   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 7 = 0x07 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x18, /*   ...XX...   */
+	0x3c, /*   ..XXXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x18, /*   ...XX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 8 = 0x08 90*/
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xe7, /*   XXX..XXX   */
+	0xc3, /*   XX....XX   */
+	0xc3, /*   XX....XX   */
+	0xe7, /*   XXX..XXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	/* 9 = 0x09 90*/
+	0x00, /*   ........   */
+	0x3c, /*   ..XXXX..   */
+	0x66, /*   .XX..XX.   */
+	0x42, /*   .X....X.   */
+	0x42, /*   .X....X.   */
+	0x66, /*   .XX..XX.   */
+	0x3c, /*   ..XXXX..   */
+	0x00, /*   ........   */
+	/* 10 = 0x0a 90*/
+	0xff, /*   XXXXXXXX   */
+	0xc3, /*   XX....XX   */
+	0x99, /*   X..XX..X   */
+	0xbd, /*   X.XXXX.X   */
+	0xbd, /*   X.XXXX.X   */
+	0x99, /*   X..XX..X   */
+	0xc3, /*   XX....XX   */
+	0xff, /*   XXXXXXXX   */
+	/* 11 = 0x0b 90*/
+	0xe0, /*   XXX.....   */
+	0xce, /*   XX..XXX.   */
+	0xff, /*   XXXXXXXX   */
+	0xb1, /*   X.XX...X   */
+	0x11, /*   ...X...X   */
+	0x1f, /*   ...XXXXX   */
+	0x0e, /*   ....XXX.   */
+	0x00, /*   ........   */
+	/* 12 = 0x0c 90*/
+	0x00, /*   ........   */
+	0x72, /*   .XXX..X.   */
+	0xfa, /*   XXXXX.X.   */
+	0x8f, /*   X...XXXX   */
+	0x8f, /*   X...XXXX   */
+	0xfa, /*   XXXXX.X.   */
+	0x72, /*   .XXX..X.   */
+	0x00, /*   ........   */
+	/* 13 = 0x0d 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x38, /*   ..XXX...   */
+	0x60, /*   .XX.....   */
+	0xfe, /*   XXXXXXX.   */
+	0xff, /*   XXXXXXXX   */
+	0x07, /*   .....XXX   */
+	0x03, /*   ......XX   */
+	/* 14 = 0x0e 90*/
+	0xfc, /*   XXXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0xa6, /*   X.X..XX.   */
+	0xa0, /*   X.X.....   */
+	0xa0, /*   X.X.....   */
+	0xfe, /*   XXXXXXX.   */
+	0xff, /*   XXXXXXXX   */
+	0x03, /*   ......XX   */
+	/* 15 = 0x0f 90*/
+	0x42, /*   .X....X.   */
+	0x7e, /*   .XXXXXX.   */
+	0x3c, /*   ..XXXX..   */
+	0xe7, /*   XXX..XXX   */
+	0xe7, /*   XXX..XXX   */
+	0x3c, /*   ..XXXX..   */
+	0x7e, /*   .XXXXXX.   */
+	0x42, /*   .X....X.   */
+	/* 16 = 0x10 90*/
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x38, /*   ..XXX...   */
+	0x38, /*   ..XXX...   */
+	0x7c, /*   .XXXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 17 = 0x11 90*/
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0x38, /*   ..XXX...   */
+	0x38, /*   ..XXX...   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 18 = 0x12 90*/
+	0x00, /*   ........   */
+	0x24, /*   ..X..X..   */
+	0x66, /*   .XX..XX.   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x66, /*   .XX..XX.   */
+	0x24, /*   ..X..X..   */
+	0x00, /*   ........   */
+	/* 19 = 0x13 90*/
+	0x00, /*   ........   */
+	0xfa, /*   XXXXX.X.   */
+	0xfa, /*   XXXXX.X.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xfa, /*   XXXXX.X.   */
+	0xfa, /*   XXXXX.X.   */
+	0x00, /*   ........   */
+	/* 20 = 0x14 90*/
+	0x80, /*   X.......   */
+	0xfe, /*   XXXXXXX.   */
+	0x80, /*   X.......   */
+	0xfe, /*   XXXXXXX.   */
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	0x60, /*   .XX.....   */
+	0x00, /*   ........   */
+	/* 21 = 0x15 90*/
+	0x40, /*   .X......   */
+	0xda, /*   XX.XX.X.   */
+	0xbf, /*   X.XXXXXX   */
+	0xa5, /*   X.X..X.X   */
+	0xfd, /*   XXXXXX.X   */
+	0x5b, /*   .X.XX.XX   */
+	0x02, /*   ......X.   */
+	0x00, /*   ........   */
+	/* 22 = 0x16 90*/
+	0x00, /*   ........   */
+	0x0e, /*   ....XXX.   */
+	0x0e, /*   ....XXX.   */
+	0x0e, /*   ....XXX.   */
+	0x0e, /*   ....XXX.   */
+	0x0e, /*   ....XXX.   */
+	0x0e, /*   ....XXX.   */
+	0x00, /*   ........   */
+	/* 23 = 0x17 90*/
+	0x00, /*   ........   */
+	0x29, /*   ..X.X..X   */
+	0x6d, /*   .XX.XX.X   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x6d, /*   .XX.XX.X   */
+	0x29, /*   ..X.X..X   */
+	0x00, /*   ........   */
+	/* 24 = 0x18 90*/
+	0x00, /*   ........   */
+	0x20, /*   ..X.....   */
+	0x60, /*   .XX.....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x60, /*   .XX.....   */
+	0x20, /*   ..X.....   */
+	0x00, /*   ........   */
+	/* 25 = 0x19 90*/
+	0x00, /*   ........   */
+	0x08, /*   ....X...   */
+	0x0c, /*   ....XX..   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x0c, /*   ....XX..   */
+	0x08, /*   ....X...   */
+	0x00, /*   ........   */
+	/* 26 = 0x1a 90*/
+	0x10, /*   ...X....   */
+	0x38, /*   ..XXX...   */
+	0x7c, /*   .XXXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 27 = 0x1b 90*/
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x7c, /*   .XXXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0x38, /*   ..XXX...   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 28 = 0x1c 90*/
+	0x04, /*   .....X..   */
+	0x04, /*   .....X..   */
+	0x04, /*   .....X..   */
+	0x04, /*   .....X..   */
+	0x04, /*   .....X..   */
+	0x3c, /*   ..XXXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x00, /*   ........   */
+	/* 29 = 0x1d 90*/
+	0x10, /*   ...X....   */
+	0x38, /*   ..XXX...   */
+	0x7c, /*   .XXXXX..   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x7c, /*   .XXXXX..   */
+	0x38, /*   ..XXX...   */
+	0x10, /*   ...X....   */
+	/* 30 = 0x1e 90*/
+	0x0c, /*   ....XX..   */
+	0x1c, /*   ...XXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x1c, /*   ...XXX..   */
+	0x0c, /*   ....XX..   */
+	/* 31 = 0x1f 90*/
+	0x60, /*   .XX.....   */
+	0x70, /*   .XXX....   */
+	0x78, /*   .XXXX...   */
+	0x7c, /*   .XXXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0x78, /*   .XXXX...   */
+	0x70, /*   .XXX....   */
+	0x60, /*   .XX.....   */
+	/* 32 = 0x20 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 33 = 0x21 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xfa, /*   XXXXX.X.   */
+	0xfa, /*   XXXXX.X.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 34 = 0x22 90*/
+	0x00, /*   ........   */
+	0xe0, /*   XXX.....   */
+	0xe0, /*   XXX.....   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xe0, /*   XXX.....   */
+	0xe0, /*   XXX.....   */
+	0x00, /*   ........   */
+	/* 35 = 0x23 90*/
+	0x00, /*   ........   */
+	0x28, /*   ..X.X...   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x28, /*   ..X.X...   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x28, /*   ..X.X...   */
+	/* 36 = 0x24 90*/
+	0x00, /*   ........   */
+	0x08, /*   ....X...   */
+	0x5c, /*   .X.XXX..   */
+	0x54, /*   .X.X.X..   */
+	0xfe, /*   XXXXXXX.   */
+	0x54, /*   .X.X.X..   */
+	0x74, /*   .XXX.X..   */
+	0x20, /*   ..X.....   */
+	/* 37 = 0x25 90*/
+	0x00, /*   ........   */
+	0x46, /*   .X...XX.   */
+	0x66, /*   .XX..XX.   */
+	0x30, /*   ..XX....   */
+	0x18, /*   ...XX...   */
+	0x0c, /*   ....XX..   */
+	0x66, /*   .XX..XX.   */
+	0x62, /*   .XX...X.   */
+	/* 38 = 0x26 90*/
+	0x00, /*   ........   */
+	0x12, /*   ...X..X.   */
+	0x5e, /*   .X.XXXX.   */
+	0xec, /*   XXX.XX..   */
+	0xba, /*   X.XXX.X.   */
+	0xf2, /*   XXXX..X.   */
+	0x5e, /*   .X.XXXX.   */
+	0x0c, /*   ....XX..   */
+	/* 39 = 0x27 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xc0, /*   XX......   */
+	0xe0, /*   XXX.....   */
+	0x20, /*   ..X.....   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 40 = 0x28 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x82, /*   X.....X.   */
+	0xc6, /*   XX...XX.   */
+	0x7c, /*   .XXXXX..   */
+	0x38, /*   ..XXX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 41 = 0x29 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x38, /*   ..XXX...   */
+	0x7c, /*   .XXXXX..   */
+	0xc6, /*   XX...XX.   */
+	0x82, /*   X.....X.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 42 = 0x2a 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x54, /*   .X.X.X..   */
+	0x7c, /*   .XXXXX..   */
+	0x38, /*   ..XXX...   */
+	0x7c, /*   .XXXXX..   */
+	0x54, /*   .X.X.X..   */
+	0x10, /*   ...X....   */
+	/* 43 = 0x2b 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x7c, /*   .XXXXX..   */
+	0x7c, /*   .XXXXX..   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 44 = 0x2c 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x06, /*   .....XX.   */
+	0x07, /*   .....XXX   */
+	0x01, /*   .......X   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 45 = 0x2d 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 46 = 0x2e 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x06, /*   .....XX.   */
+	0x06, /*   .....XX.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 47 = 0x2f 90*/
+	0x00, /*   ........   */
+	0x40, /*   .X......   */
+	0x60, /*   .XX.....   */
+	0x30, /*   ..XX....   */
+	0x18, /*   ...XX...   */
+	0x0c, /*   ....XX..   */
+	0x06, /*   .....XX.   */
+	0x02, /*   ......X.   */
+	/* 48 = 0x30 90*/
+	0x00, /*   ........   */
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0xb2, /*   X.XX..X.   */
+	0x9a, /*   X..XX.X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x00, /*   ........   */
+	/* 49 = 0x31 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x60, /*   .XX.....   */
+	0x20, /*   ..X.....   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 50 = 0x32 90*/
+	0x00, /*   ........   */
+	0x62, /*   .XX...X.   */
+	0xf2, /*   XXXX..X.   */
+	0x92, /*   X..X..X.   */
+	0x9a, /*   X..XX.X.   */
+	0xce, /*   XX..XXX.   */
+	0x46, /*   .X...XX.   */
+	0x00, /*   ........   */
+	/* 51 = 0x33 90*/
+	0x00, /*   ........   */
+	0x6c, /*   .XX.XX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xc6, /*   XX...XX.   */
+	0x44, /*   .X...X..   */
+	0x00, /*   ........   */
+	/* 52 = 0x34 90*/
+	0x00, /*   ........   */
+	0x08, /*   ....X...   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0xc8, /*   XX..X...   */
+	0x68, /*   .XX.X...   */
+	0x38, /*   ..XXX...   */
+	0x18, /*   ...XX...   */
+	/* 53 = 0x35 90*/
+	0x00, /*   ........   */
+	0x8c, /*   X...XX..   */
+	0x9e, /*   X..XXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xf6, /*   XXXX.XX.   */
+	0xf4, /*   XXXX.X..   */
+	0x00, /*   ........   */
+	/* 54 = 0x36 90*/
+	0x00, /*   ........   */
+	0x0c, /*   ....XX..   */
+	0x9e, /*   X..XXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x00, /*   ........   */
+	/* 55 = 0x37 90*/
+	0x00, /*   ........   */
+	0xe0, /*   XXX.....   */
+	0xf0, /*   XXXX....   */
+	0x98, /*   X..XX...   */
+	0x8e, /*   X...XXX.   */
+	0x86, /*   X....XX.   */
+	0x80, /*   X.......   */
+	0x00, /*   ........   */
+	/* 56 = 0x38 90*/
+	0x00, /*   ........   */
+	0x6c, /*   .XX.XX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x6c, /*   .XX.XX..   */
+	0x00, /*   ........   */
+	/* 57 = 0x39 90*/
+	0x00, /*   ........   */
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xf2, /*   XXXX..X.   */
+	0x60, /*   .XX.....   */
+	0x00, /*   ........   */
+	/* 58 = 0x3a 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x66, /*   .XX..XX.   */
+	0x66, /*   .XX..XX.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 59 = 0x3b 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x66, /*   .XX..XX.   */
+	0x67, /*   .XX..XXX   */
+	0x01, /*   .......X   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 60 = 0x3c 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x82, /*   X.....X.   */
+	0xc6, /*   XX...XX.   */
+	0x6c, /*   .XX.XX..   */
+	0x38, /*   ..XXX...   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 61 = 0x3d 90*/
+	0x00, /*   ........   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x00, /*   ........   */
+	/* 62 = 0x3e 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x38, /*   ..XXX...   */
+	0x6c, /*   .XX.XX..   */
+	0xc6, /*   XX...XX.   */
+	0x82, /*   X.....X.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 63 = 0x3f 90*/
+	0x00, /*   ........   */
+	0x60, /*   .XX.....   */
+	0xf0, /*   XXXX....   */
+	0x9a, /*   X..XX.X.   */
+	0x8a, /*   X...X.X.   */
+	0xc0, /*   XX......   */
+	0x40, /*   .X......   */
+	0x00, /*   ........   */
+	/* 64 = 0x40 90*/
+	0x00, /*   ........   */
+	0x70, /*   .XXX....   */
+	0xfa, /*   XXXXX.X.   */
+	0xba, /*   X.XXX.X.   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x00, /*   ........   */
+	/* 65 = 0x41 90*/
+	0x00, /*   ........   */
+	0x3e, /*   ..XXXXX.   */
+	0x7e, /*   .XXXXXX.   */
+	0xc8, /*   XX..X...   */
+	0xc8, /*   XX..X...   */
+	0x7e, /*   .XXXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x00, /*   ........   */
+	/* 66 = 0x42 90*/
+	0x00, /*   ........   */
+	0x6c, /*   .XX.XX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 67 = 0x43 90*/
+	0x00, /*   ........   */
+	0x44, /*   .X...X..   */
+	0xc6, /*   XX...XX.   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x00, /*   ........   */
+	/* 68 = 0x44 90*/
+	0x00, /*   ........   */
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 69 = 0x45 90*/
+	0x00, /*   ........   */
+	0x82, /*   X.....X.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 70 = 0x46 90*/
+	0x00, /*   ........   */
+	0x80, /*   X.......   */
+	0x90, /*   X..X....   */
+	0x90, /*   X..X....   */
+	0x90, /*   X..X....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 71 = 0x47 90*/
+	0x00, /*   ........   */
+	0x5c, /*   .X.XXX..   */
+	0xde, /*   XX.XXXX.   */
+	0x92, /*   X..X..X.   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x00, /*   ........   */
+	/* 72 = 0x48 90*/
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 73 = 0x49 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x82, /*   X.....X.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 74 = 0x4a 90*/
+	0x00, /*   ........   */
+	0xfc, /*   XXXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0x0e, /*   ....XXX.   */
+	0x0c, /*   ....XX..   */
+	0x00, /*   ........   */
+	/* 75 = 0x4b 90*/
+	0x00, /*   ........   */
+	0x82, /*   X.....X.   */
+	0xc6, /*   XX...XX.   */
+	0x6c, /*   .XX.XX..   */
+	0x38, /*   ..XXX...   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 76 = 0x4c 90*/
+	0x00, /*   ........   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 77 = 0x4d 90*/
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x60, /*   .XX.....   */
+	0x30, /*   ..XX....   */
+	0x60, /*   .XX.....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	/* 78 = 0x4e 90*/
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x30, /*   ..XX....   */
+	0x60, /*   .XX.....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 79 = 0x4f 90*/
+	0x00, /*   ........   */
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x00, /*   ........   */
+	/* 80 = 0x50 90*/
+	0x00, /*   ........   */
+	0x60, /*   .XX.....   */
+	0xf0, /*   XXXX....   */
+	0x90, /*   X..X....   */
+	0x90, /*   X..X....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 81 = 0x51 90*/
+	0x00, /*   ........   */
+	0x7d, /*   .XXXXX.X   */
+	0xff, /*   XXXXXXXX   */
+	0x86, /*   X....XX.   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x00, /*   ........   */
+	/* 82 = 0x52 90*/
+	0x00, /*   ........   */
+	0x6e, /*   .XX.XXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x90, /*   X..X....   */
+	0x90, /*   X..X....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 83 = 0x53 90*/
+	0x00, /*   ........   */
+	0x4c, /*   .X..XX..   */
+	0xde, /*   XX.XXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xf6, /*   XXXX.XX.   */
+	0x64, /*   .XX..X..   */
+	0x00, /*   ........   */
+	/* 84 = 0x54 90*/
+	0x00, /*   ........   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0x00, /*   ........   */
+	/* 85 = 0x55 90*/
+	0x00, /*   ........   */
+	0xfc, /*   XXXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfc, /*   XXXXXX..   */
+	0x00, /*   ........   */
+	/* 86 = 0x56 90*/
+	0x00, /*   ........   */
+	0xf8, /*   XXXXX...   */
+	0xfc, /*   XXXXXX..   */
+	0x06, /*   .....XX.   */
+	0x06, /*   .....XX.   */
+	0xfc, /*   XXXXXX..   */
+	0xf8, /*   XXXXX...   */
+	0x00, /*   ........   */
+	/* 87 = 0x57 90*/
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x0c, /*   ....XX..   */
+	0x18, /*   ...XX...   */
+	0x0c, /*   ....XX..   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	/* 88 = 0x58 90*/
+	0x00, /*   ........   */
+	0xc6, /*   XX...XX.   */
+	0xee, /*   XXX.XXX.   */
+	0x38, /*   ..XXX...   */
+	0x38, /*   ..XXX...   */
+	0xee, /*   XXX.XXX.   */
+	0xc6, /*   XX...XX.   */
+	0x00, /*   ........   */
+	/* 89 = 0x59 90*/
+	0x00, /*   ........   */
+	0xe0, /*   XXX.....   */
+	0xf0, /*   XXXX....   */
+	0x1e, /*   ...XXXX.   */
+	0x1e, /*   ...XXXX.   */
+	0xf0, /*   XXXX....   */
+	0xe0, /*   XXX.....   */
+	0x00, /*   ........   */
+	/* 90 = 0x5a 90*/
+	0x00, /*   ........   */
+	0xc2, /*   XX....X.   */
+	0xe2, /*   XXX...X.   */
+	0xb2, /*   X.XX..X.   */
+	0x9a, /*   X..XX.X.   */
+	0x8e, /*   X...XXX.   */
+	0x86, /*   X....XX.   */
+	0x00, /*   ........   */
+	/* 91 = 0x5b 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 92 = 0x5c 90*/
+	0x00, /*   ........   */
+	0x02, /*   ......X.   */
+	0x06, /*   .....XX.   */
+	0x0c, /*   ....XX..   */
+	0x18, /*   ...XX...   */
+	0x30, /*   ..XX....   */
+	0x60, /*   .XX.....   */
+	0x40, /*   .X......   */
+	/* 93 = 0x5d 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 94 = 0x5e 90*/
+	0x00, /*   ........   */
+	0x20, /*   ..X.....   */
+	0x60, /*   .XX.....   */
+	0xc0, /*   XX......   */
+	0xc0, /*   XX......   */
+	0x60, /*   .XX.....   */
+	0x20, /*   ..X.....   */
+	0x00, /*   ........   */
+	/* 95 = 0x5f 90*/
+	0x01, /*   .......X   */
+	0x01, /*   .......X   */
+	0x01, /*   .......X   */
+	0x01, /*   .......X   */
+	0x01, /*   .......X   */
+	0x01, /*   .......X   */
+	0x01, /*   .......X   */
+	0x01, /*   .......X   */
+	/* 96 = 0x60 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x20, /*   ..X.....   */
+	0xe0, /*   XXX.....   */
+	0xc0, /*   XX......   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 97 = 0x61 90*/
+	0x00, /*   ........   */
+	0x1e, /*   ...XXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2e, /*   ..X.XXX.   */
+	0x04, /*   .....X..   */
+	0x00, /*   ........   */
+	/* 98 = 0x62 90*/
+	0x00, /*   ........   */
+	0x0c, /*   ....XX..   */
+	0x1e, /*   ...XXXX.   */
+	0x12, /*   ...X..X.   */
+	0x12, /*   ...X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 99 = 0x63 90*/
+	0x00, /*   ........   */
+	0x14, /*   ...X.X..   */
+	0x36, /*   ..XX.XX.   */
+	0x22, /*   ..X...X.   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 100 = 0x64 90*/
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x12, /*   ...X..X.   */
+	0x12, /*   ...X..X.   */
+	0x1e, /*   ...XXXX.   */
+	0x0c, /*   ....XX..   */
+	0x00, /*   ........   */
+	/* 101 = 0x65 90*/
+	0x00, /*   ........   */
+	0x1a, /*   ...XX.X.   */
+	0x3a, /*   ..XXX.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 102 = 0x66 90*/
+	0x00, /*   ........   */
+	0x40, /*   .X......   */
+	0xd0, /*   XX.X....   */
+	0x90, /*   X..X....   */
+	0xfe, /*   XXXXXXX.   */
+	0x7e, /*   .XXXXXX.   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 103 = 0x67 90*/
+	0x00, /*   ........   */
+	0x3e, /*   ..XXXXX.   */
+	0x3f, /*   ..XXXXXX   */
+	0x25, /*   ..X..X.X   */
+	0x25, /*   ..X..X.X   */
+	0x3d, /*   ..XXXX.X   */
+	0x18, /*   ...XX...   */
+	0x00, /*   ........   */
+	/* 104 = 0x68 90*/
+	0x00, /*   ........   */
+	0x1e, /*   ...XXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x20, /*   ..X.....   */
+	0x20, /*   ..X.....   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 105 = 0x69 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xbe, /*   X.XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 106 = 0x6a 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xbe, /*   X.XXXXX.   */
+	0xbf, /*   X.XXXXXX   */
+	0x01, /*   .......X   */
+	0x03, /*   ......XX   */
+	0x02, /*   ......X.   */
+	0x00, /*   ........   */
+	/* 107 = 0x6b 90*/
+	0x00, /*   ........   */
+	0x22, /*   ..X...X.   */
+	0x36, /*   ..XX.XX.   */
+	0x1c, /*   ...XXX..   */
+	0x08, /*   ....X...   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 108 = 0x6c 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x02, /*   ......X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfc, /*   XXXXXX..   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 109 = 0x6d 90*/
+	0x00, /*   ........   */
+	0x1e, /*   ...XXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x30, /*   ..XX....   */
+	0x1c, /*   ...XXX..   */
+	0x30, /*   ..XX....   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	/* 110 = 0x6e 90*/
+	0x00, /*   ........   */
+	0x1e, /*   ...XXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x20, /*   ..X.....   */
+	0x20, /*   ..X.....   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x00, /*   ........   */
+	/* 111 = 0x6f 90*/
+	0x00, /*   ........   */
+	0x1c, /*   ...XXX..   */
+	0x3e, /*   ..XXXXX.   */
+	0x22, /*   ..X...X.   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 112 = 0x70 90*/
+	0x00, /*   ........   */
+	0x18, /*   ...XX...   */
+	0x3c, /*   ..XXXX..   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x3f, /*   ..XXXXXX   */
+	0x3f, /*   ..XXXXXX   */
+	0x00, /*   ........   */
+	/* 113 = 0x71 90*/
+	0x00, /*   ........   */
+	0x3f, /*   ..XXXXXX   */
+	0x3f, /*   ..XXXXXX   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x3c, /*   ..XXXX..   */
+	0x18, /*   ...XX...   */
+	0x00, /*   ........   */
+	/* 114 = 0x72 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x30, /*   ..XX....   */
+	0x20, /*   ..X.....   */
+	0x20, /*   ..X.....   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x00, /*   ........   */
+	/* 115 = 0x73 90*/
+	0x00, /*   ........   */
+	0x24, /*   ..X..X..   */
+	0x2e, /*   ..X.XXX.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x3a, /*   ..XXX.X.   */
+	0x12, /*   ...X..X.   */
+	0x00, /*   ........   */
+	/* 116 = 0x74 90*/
+	0x00, /*   ........   */
+	0x04, /*   .....X..   */
+	0x26, /*   ..X..XX.   */
+	0x22, /*   ..X...X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfc, /*   XXXXXX..   */
+	0x20, /*   ..X.....   */
+	0x00, /*   ........   */
+	/* 117 = 0x75 90*/
+	0x00, /*   ........   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3c, /*   ..XXXX..   */
+	0x00, /*   ........   */
+	/* 118 = 0x76 90*/
+	0x00, /*   ........   */
+	0x38, /*   ..XXX...   */
+	0x3c, /*   ..XXXX..   */
+	0x06, /*   .....XX.   */
+	0x06, /*   .....XX.   */
+	0x3c, /*   ..XXXX..   */
+	0x38, /*   ..XXX...   */
+	0x00, /*   ........   */
+	/* 119 = 0x77 90*/
+	0x00, /*   ........   */
+	0x3c, /*   ..XXXX..   */
+	0x3e, /*   ..XXXXX.   */
+	0x0c, /*   ....XX..   */
+	0x18, /*   ...XX...   */
+	0x0c, /*   ....XX..   */
+	0x3e, /*   ..XXXXX.   */
+	0x3c, /*   ..XXXX..   */
+	/* 120 = 0x78 90*/
+	0x00, /*   ........   */
+	0x22, /*   ..X...X.   */
+	0x36, /*   ..XX.XX.   */
+	0x1c, /*   ...XXX..   */
+	0x1c, /*   ...XXX..   */
+	0x36, /*   ..XX.XX.   */
+	0x22, /*   ..X...X.   */
+	0x00, /*   ........   */
+	/* 121 = 0x79 90*/
+	0x00, /*   ........   */
+	0x3e, /*   ..XXXXX.   */
+	0x3f, /*   ..XXXXXX   */
+	0x05, /*   .....X.X   */
+	0x05, /*   .....X.X   */
+	0x3d, /*   ..XXXX.X   */
+	0x38, /*   ..XXX...   */
+	0x00, /*   ........   */
+	/* 122 = 0x7a 90*/
+	0x00, /*   ........   */
+	0x22, /*   ..X...X.   */
+	0x32, /*   ..XX..X.   */
+	0x3a, /*   ..XXX.X.   */
+	0x2e, /*   ..X.XXX.   */
+	0x26, /*   ..X..XX.   */
+	0x22, /*   ..X...X.   */
+	0x00, /*   ........   */
+	/* 123 = 0x7b 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0xee, /*   XXX.XXX.   */
+	0x7c, /*   .XXXXX..   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 124 = 0x7c 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 125 = 0x7d 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x7c, /*   .XXXXX..   */
+	0xee, /*   XXX.XXX.   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 126 = 0x7e 90*/
+	0x00, /*   ........   */
+	0x80, /*   X.......   */
+	0xc0, /*   XX......   */
+	0x40, /*   .X......   */
+	0xc0, /*   XX......   */
+	0x80, /*   X.......   */
+	0xc0, /*   XX......   */
+	0x40, /*   .X......   */
+	/* 127 = 0x7f 90*/
+	0x00, /*   ........   */
+	0x0e, /*   ....XXX.   */
+	0x1e, /*   ...XXXX.   */
+	0x32, /*   ..XX..X.   */
+	0x32, /*   ..XX..X.   */
+	0x1e, /*   ...XXXX.   */
+	0x0e, /*   ....XXX.   */
+	0x00, /*   ........   */
+	/* 128 = 0x80 90*/
+	0x00, /*   ........   */
+	0x48, /*   .X..X...   */
+	0xcc, /*   XX..XX..   */
+	0x86, /*   X....XX.   */
+	0x87, /*   X....XXX   */
+	0xfd, /*   XXXXXX.X   */
+	0x78, /*   .XXXX...   */
+	0x00, /*   ........   */
+	/* 129 = 0x81 90*/
+	0x00, /*   ........   */
+	0xbe, /*   X.XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0xbe, /*   X.XXXXX.   */
+	0xbc, /*   X.XXXX..   */
+	0x00, /*   ........   */
+	/* 130 = 0x82 90*/
+	0x00, /*   ........   */
+	0x98, /*   X..XX...   */
+	0xba, /*   X.XXX.X.   */
+	0xaa, /*   X.X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 131 = 0x83 90*/
+	0x00, /*   ........   */
+	0x5e, /*   .X.XXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0xaa, /*   X.X.X.X.   */
+	0xaa, /*   X.X.X.X.   */
+	0xee, /*   XXX.XXX.   */
+	0x44, /*   .X...X..   */
+	0x00, /*   ........   */
+	/* 132 = 0x84 90*/
+	0x00, /*   ........   */
+	0x9e, /*   X..XXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0xae, /*   X.X.XXX.   */
+	0x84, /*   X....X..   */
+	0x00, /*   ........   */
+	/* 133 = 0x85 90*/
+	0x00, /*   ........   */
+	0x1e, /*   ...XXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x2a, /*   ..X.X.X.   */
+	0xaa, /*   X.X.X.X.   */
+	0xae, /*   X.X.XXX.   */
+	0x84, /*   X....X..   */
+	0x00, /*   ........   */
+	/* 134 = 0x86 90*/
+	0x00, /*   ........   */
+	0x1e, /*   ...XXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0xea, /*   XXX.X.X.   */
+	0xea, /*   XXX.X.X.   */
+	0x2e, /*   ..X.XXX.   */
+	0x04, /*   .....X..   */
+	0x00, /*   ........   */
+	/* 135 = 0x87 90*/
+	0x00, /*   ........   */
+	0x24, /*   ..X..X..   */
+	0x24, /*   ..X..X..   */
+	0x26, /*   ..X..XX.   */
+	0x27, /*   ..X..XXX   */
+	0x3d, /*   ..XXXX.X   */
+	0x18, /*   ...XX...   */
+	0x00, /*   ........   */
+	/* 136 = 0x88 90*/
+	0x00, /*   ........   */
+	0x58, /*   .X.XX...   */
+	0xfa, /*   XXXXX.X.   */
+	0xaa, /*   X.X.X.X.   */
+	0xaa, /*   X.X.X.X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x5c, /*   .X.XXX..   */
+	0x00, /*   ........   */
+	/* 137 = 0x89 90*/
+	0x00, /*   ........   */
+	0x98, /*   X..XX...   */
+	0xba, /*   X.XXX.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0xbe, /*   X.XXXXX.   */
+	0x9c, /*   X..XXX..   */
+	0x00, /*   ........   */
+	/* 138 = 0x8a 90*/
+	0x00, /*   ........   */
+	0x18, /*   ...XX...   */
+	0x3a, /*   ..XXX.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0xaa, /*   X.X.X.X.   */
+	0xbe, /*   X.XXXXX.   */
+	0x9c, /*   X..XXX..   */
+	0x00, /*   ........   */
+	/* 139 = 0x8b 90*/
+	0x00, /*   ........   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0x00, /*   ........   */
+	/* 140 = 0x8c 90*/
+	0x00, /*   ........   */
+	0x40, /*   .X......   */
+	0xc0, /*   XX......   */
+	0xbe, /*   X.XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0xc0, /*   XX......   */
+	0x40, /*   .X......   */
+	0x00, /*   ........   */
+	/* 141 = 0x8d 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x3e, /*   ..XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0x00, /*   ........   */
+	/* 142 = 0x8e 90*/
+	0x00, /*   ........   */
+	0x9e, /*   X..XXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x68, /*   .XX.X...   */
+	0x48, /*   .X..X...   */
+	0x68, /*   .XX.X...   */
+	0xbe, /*   X.XXXXX.   */
+	0x9e, /*   X..XXXX.   */
+	/* 143 = 0x8f 90*/
+	0x00, /*   ........   */
+	0x0e, /*   ....XXX.   */
+	0x1e, /*   ...XXXX.   */
+	0xd4, /*   XX.X.X..   */
+	0xd4, /*   XX.X.X..   */
+	0x1e, /*   ...XXXX.   */
+	0x0e, /*   ....XXX.   */
+	0x00, /*   ........   */
+	/* 144 = 0x90 90*/
+	0x00, /*   ........   */
+	0x80, /*   X.......   */
+	0xa2, /*   X.X...X.   */
+	0xaa, /*   X.X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x00, /*   ........   */
+	/* 145 = 0x91 90*/
+	0x00, /*   ........   */
+	0x3a, /*   ..XXX.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2e, /*   ..X.XXX.   */
+	0x04, /*   .....X..   */
+	/* 146 = 0x92 90*/
+	0x00, /*   ........   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0xc8, /*   XX..X...   */
+	0x7e, /*   .XXXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	/* 147 = 0x93 90*/
+	0x00, /*   ........   */
+	0x5c, /*   .X.XXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0xa2, /*   X.X...X.   */
+	0xa2, /*   X.X...X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x5c, /*   .X.XXX..   */
+	0x00, /*   ........   */
+	/* 148 = 0x94 90*/
+	0x00, /*   ........   */
+	0x9c, /*   X..XXX..   */
+	0xbe, /*   X.XXXXX.   */
+	0x22, /*   ..X...X.   */
+	0x22, /*   ..X...X.   */
+	0xbe, /*   X.XXXXX.   */
+	0x9c, /*   X..XXX..   */
+	0x00, /*   ........   */
+	/* 149 = 0x95 90*/
+	0x00, /*   ........   */
+	0x1c, /*   ...XXX..   */
+	0x3e, /*   ..XXXXX.   */
+	0x22, /*   ..X...X.   */
+	0xa2, /*   X.X...X.   */
+	0xbe, /*   X.XXXXX.   */
+	0x9c, /*   X..XXX..   */
+	0x00, /*   ........   */
+	/* 150 = 0x96 90*/
+	0x00, /*   ........   */
+	0x5e, /*   .X.XXXX.   */
+	0xde, /*   XX.XXXX.   */
+	0x82, /*   X.....X.   */
+	0x82, /*   X.....X.   */
+	0xde, /*   XX.XXXX.   */
+	0x5c, /*   .X.XXX..   */
+	0x00, /*   ........   */
+	/* 151 = 0x97 90*/
+	0x00, /*   ........   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x02, /*   ......X.   */
+	0x82, /*   X.....X.   */
+	0xbe, /*   X.XXXXX.   */
+	0xbc, /*   X.XXXX..   */
+	0x00, /*   ........   */
+	/* 152 = 0x98 90*/
+	0x00, /*   ........   */
+	0xbe, /*   X.XXXXX.   */
+	0xbf, /*   X.XXXXXX   */
+	0x05, /*   .....X.X   */
+	0x05, /*   .....X.X   */
+	0xbd, /*   X.XXXX.X   */
+	0xb8, /*   X.XXX...   */
+	0x00, /*   ........   */
+	/* 153 = 0x99 90*/
+	0x00, /*   ........   */
+	0xbc, /*   X.XXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x42, /*   .X....X.   */
+	0x42, /*   .X....X.   */
+	0xfe, /*   XXXXXXX.   */
+	0xbc, /*   X.XXXX..   */
+	0x00, /*   ........   */
+	/* 154 = 0x9a 90*/
+	0x00, /*   ........   */
+	0xbc, /*   X.XXXX..   */
+	0xbe, /*   X.XXXXX.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0xbe, /*   X.XXXXX.   */
+	0xbc, /*   X.XXXX..   */
+	0x00, /*   ........   */
+	/* 155 = 0x9b 90*/
+	0x00, /*   ........   */
+	0x24, /*   ..X..X..   */
+	0xe7, /*   XXX..XXX   */
+	0xe7, /*   XXX..XXX   */
+	0x24, /*   ..X..X..   */
+	0x3c, /*   ..XXXX..   */
+	0x18, /*   ...XX...   */
+	0x00, /*   ........   */
+	/* 156 = 0x9c 90*/
+	0x00, /*   ........   */
+	0x04, /*   .....X..   */
+	0x46, /*   .X...XX.   */
+	0xc2, /*   XX....X.   */
+	0x92, /*   X..X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7e, /*   .XXXXXX.   */
+	0x12, /*   ...X..X.   */
+	/* 157 = 0x9d 90*/
+	0x00, /*   ........   */
+	0xca, /*   XX..X.X.   */
+	0xea, /*   XXX.X.X.   */
+	0x3f, /*   ..XXXXXX   */
+	0x3f, /*   ..XXXXXX   */
+	0xea, /*   XXX.X.X.   */
+	0xca, /*   XX..X.X.   */
+	0x00, /*   ........   */
+	/* 158 = 0x9e 90*/
+	0x05, /*   .....X.X   */
+	0x6f, /*   .XX.XXXX   */
+	0xfe, /*   XXXXXXX.   */
+	0x94, /*   X..X.X..   */
+	0x90, /*   X..X....   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	/* 159 = 0x9f 90*/
+	0x40, /*   .X......   */
+	0xc0, /*   XX......   */
+	0x90, /*   X..X....   */
+	0xfe, /*   XXXXXXX.   */
+	0x7f, /*   .XXXXXXX   */
+	0x13, /*   ...X..XX   */
+	0x02, /*   ......X.   */
+	0x00, /*   ........   */
+	/* 160 = 0xa0 90*/
+	0x00, /*   ........   */
+	0x9e, /*   X..XXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0xaa, /*   X.X.X.X.   */
+	0x2a, /*   ..X.X.X.   */
+	0x2e, /*   ..X.XXX.   */
+	0x04, /*   .....X..   */
+	0x00, /*   ........   */
+	/* 161 = 0xa1 90*/
+	0x00, /*   ........   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0xbe, /*   X.XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 162 = 0xa2 90*/
+	0x00, /*   ........   */
+	0x9c, /*   X..XXX..   */
+	0xbe, /*   X.XXXXX.   */
+	0xa2, /*   X.X...X.   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 163 = 0xa3 90*/
+	0x00, /*   ........   */
+	0xbe, /*   X.XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x82, /*   X.....X.   */
+	0x02, /*   ......X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3c, /*   ..XXXX..   */
+	0x00, /*   ........   */
+	/* 164 = 0xa4 90*/
+	0x00, /*   ........   */
+	0x8e, /*   X...XXX.   */
+	0xde, /*   XX.XXXX.   */
+	0x50, /*   .X.X....   */
+	0xd0, /*   XX.X....   */
+	0x9e, /*   X..XXXX.   */
+	0xde, /*   XX.XXXX.   */
+	0x40, /*   .X......   */
+	/* 165 = 0xa5 90*/
+	0x00, /*   ........   */
+	0xbe, /*   X.XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x8c, /*   X...XX..   */
+	0x98, /*   X..XX...   */
+	0xbe, /*   X.XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x00, /*   ........   */
+	/* 166 = 0xa6 90*/
+	0x00, /*   ........   */
+	0xf4, /*   XXXX.X..   */
+	0xf4, /*   XXXX.X..   */
+	0x94, /*   X..X.X..   */
+	0x94, /*   X..X.X..   */
+	0xf4, /*   XXXX.X..   */
+	0x64, /*   .XX..X..   */
+	0x00, /*   ........   */
+	/* 167 = 0xa7 90*/
+	0x00, /*   ........   */
+	0x64, /*   .XX..X..   */
+	0xf4, /*   XXXX.X..   */
+	0x94, /*   X..X.X..   */
+	0x94, /*   X..X.X..   */
+	0xf4, /*   XXXX.X..   */
+	0x64, /*   .XX..X..   */
+	0x00, /*   ........   */
+	/* 168 = 0xa8 90*/
+	0x00, /*   ........   */
+	0x04, /*   .....X..   */
+	0x06, /*   .....XX.   */
+	0xa2, /*   X.X...X.   */
+	0xb2, /*   X.XX..X.   */
+	0x1e, /*   ...XXXX.   */
+	0x0c, /*   ....XX..   */
+	0x00, /*   ........   */
+	/* 169 = 0xa9 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x1c, /*   ...XXX..   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 170 = 0xaa 90*/
+	0x00, /*   ........   */
+	0x1c, /*   ...XXX..   */
+	0x1c, /*   ...XXX..   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 171 = 0xab 90*/
+	0x09, /*   ....X..X   */
+	0x9d, /*   X..XXX.X   */
+	0xd7, /*   XX.X.XXX   */
+	0x73, /*   .XXX..XX   */
+	0x31, /*   ..XX...X   */
+	0x18, /*   ...XX...   */
+	0xec, /*   XXX.XX..   */
+	0xe4, /*   XXX..X..   */
+	/* 172 = 0xac 90*/
+	0x02, /*   ......X.   */
+	0x9f, /*   X..XXXXX   */
+	0xdf, /*   XX.XXXXX   */
+	0x6a, /*   .XX.X.X.   */
+	0x36, /*   ..XX.XX.   */
+	0x18, /*   ...XX...   */
+	0xec, /*   XXX.XX..   */
+	0xe4, /*   XXX..X..   */
+	/* 173 = 0xad 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xbe, /*   X.XXXXX.   */
+	0xbe, /*   X.XXXXX.   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 174 = 0xae 90*/
+	0x00, /*   ........   */
+	0x44, /*   .X...X..   */
+	0x6c, /*   .XX.XX..   */
+	0x38, /*   ..XXX...   */
+	0x54, /*   .X.X.X..   */
+	0x6c, /*   .XX.XX..   */
+	0x38, /*   ..XXX...   */
+	0x10, /*   ...X....   */
+	/* 175 = 0xaf 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x38, /*   ..XXX...   */
+	0x6c, /*   .XX.XX..   */
+	0x54, /*   .X.X.X..   */
+	0x38, /*   ..XXX...   */
+	0x6c, /*   .XX.XX..   */
+	0x44, /*   .X...X..   */
+	/* 176 = 0xb0 90*/
+	0x00, /*   ........   */
+	0xaa, /*   X.X.X.X.   */
+	0x00, /*   ........   */
+	0x55, /*   .X.X.X.X   */
+	0x00, /*   ........   */
+	0xaa, /*   X.X.X.X.   */
+	0x00, /*   ........   */
+	0x55, /*   .X.X.X.X   */
+	/* 177 = 0xb1 90*/
+	0xaa, /*   X.X.X.X.   */
+	0x55, /*   .X.X.X.X   */
+	0xaa, /*   X.X.X.X.   */
+	0x55, /*   .X.X.X.X   */
+	0xaa, /*   X.X.X.X.   */
+	0x55, /*   .X.X.X.X   */
+	0xaa, /*   X.X.X.X.   */
+	0x55, /*   .X.X.X.X   */
+	/* 178 = 0xb2 90*/
+	0xff, /*   XXXXXXXX   */
+	0x55, /*   .X.X.X.X   */
+	0xff, /*   XXXXXXXX   */
+	0xaa, /*   X.X.X.X.   */
+	0xff, /*   XXXXXXXX   */
+	0x55, /*   .X.X.X.X   */
+	0xff, /*   XXXXXXXX   */
+	0xaa, /*   X.X.X.X.   */
+	/* 179 = 0xb3 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 180 = 0xb4 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xff, /*   XXXXXXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 181 = 0xb5 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xff, /*   XXXXXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 182 = 0xb6 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 183 = 0xb7 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 184 = 0xb8 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x1f, /*   ...XXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 185 = 0xb9 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 186 = 0xba 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 187 = 0xbb 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x1f, /*   ...XXXXX   */
+	0x1f, /*   ...XXXXX   */
+	0x1f, /*   ...XXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 188 = 0xbc 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 189 = 0xbd 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 190 = 0xbe 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xf8, /*   XXXXX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 191 = 0xbf 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x0f, /*   ....XXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 192 = 0xc0 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xf8, /*   XXXXX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 193 = 0xc1 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xf8, /*   XXXXX...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 194 = 0xc2 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x0f, /*   ....XXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 195 = 0xc3 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 196 = 0xc4 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 197 = 0xc5 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xff, /*   XXXXXXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 198 = 0xc6 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 199 = 0xc7 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 200 = 0xc8 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 201 = 0xc9 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x1f, /*   ...XXXXX   */
+	0x1f, /*   ...XXXXX   */
+	0x1f, /*   ...XXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 202 = 0xca 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 203 = 0xcb 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x1f, /*   ...XXXXX   */
+	0x1f, /*   ...XXXXX   */
+	0x1f, /*   ...XXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 204 = 0xcc 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 205 = 0xcd 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 206 = 0xce 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 207 = 0xcf 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xf8, /*   XXXXX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 208 = 0xd0 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 209 = 0xd1 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x1f, /*   ...XXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 210 = 0xd2 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 211 = 0xd3 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 212 = 0xd4 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xf8, /*   XXXXX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 213 = 0xd5 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x1f, /*   ...XXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 214 = 0xd6 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 215 = 0xd7 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 216 = 0xd8 90*/
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0xff, /*   XXXXXXXX   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	/* 217 = 0xd9 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xf8, /*   XXXXX...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	/* 218 = 0xda 90*/
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x0f, /*   ....XXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 219 = 0xdb 90*/
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	/* 220 = 0xdc 90*/
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	0x0f, /*   ....XXXX   */
+	/* 221 = 0xdd 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	/* 222 = 0xde 90*/
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 223 = 0xdf 90*/
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	0xf0, /*   XXXX....   */
+	/* 224 = 0xe0 90*/
+	0x00, /*   ........   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	0x22, /*   ..X...X.   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	/* 225 = 0xe1 90*/
+	0x00, /*   ........   */
+	0x6c, /*   .XX.XX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xff, /*   XXXXXXXX   */
+	0x7f, /*   .XXXXXXX   */
+	0x00, /*   ........   */
+	/* 226 = 0xe2 90*/
+	0x00, /*   ........   */
+	0xc0, /*   XX......   */
+	0xc0, /*   XX......   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0xfe, /*   XXXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x00, /*   ........   */
+	/* 227 = 0xe3 90*/
+	0x00, /*   ........   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3c, /*   ..XXXX..   */
+	0x20, /*   ..X.....   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x20, /*   ..X.....   */
+	/* 228 = 0xe4 90*/
+	0x00, /*   ........   */
+	0xc6, /*   XX...XX.   */
+	0xc6, /*   XX...XX.   */
+	0x92, /*   X..X..X.   */
+	0xba, /*   X.XXX.X.   */
+	0xee, /*   XXX.XXX.   */
+	0xc6, /*   XX...XX.   */
+	0x00, /*   ........   */
+	/* 229 = 0xe5 90*/
+	0x00, /*   ........   */
+	0x20, /*   ..X.....   */
+	0x3c, /*   ..XXXX..   */
+	0x3e, /*   ..XXXXX.   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x1c, /*   ...XXX..   */
+	0x00, /*   ........   */
+	/* 230 = 0xe6 90*/
+	0x02, /*   ......X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3e, /*   ..XXXXX.   */
+	0x02, /*   ......X.   */
+	0x02, /*   ......X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3f, /*   ..XXXXXX   */
+	0x01, /*   .......X   */
+	/* 231 = 0xe7 90*/
+	0x00, /*   ........   */
+	0x20, /*   ..X.....   */
+	0x22, /*   ..X...X.   */
+	0x3e, /*   ..XXXXX.   */
+	0x3c, /*   ..XXXX..   */
+	0x20, /*   ..X.....   */
+	0x30, /*   ..XX....   */
+	0x10, /*   ...X....   */
+	/* 232 = 0xe8 90*/
+	0x00, /*   ........   */
+	0x18, /*   ...XX...   */
+	0xbd, /*   X.XXXX.X   */
+	0xe7, /*   XXX..XXX   */
+	0xff, /*   XXXXXXXX   */
+	0xe7, /*   XXX..XXX   */
+	0xbd, /*   X.XXXX.X   */
+	0x18, /*   ...XX...   */
+	/* 233 = 0xe9 90*/
+	0x00, /*   ........   */
+	0x7c, /*   .XXXXX..   */
+	0xfe, /*   XXXXXXX.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x7c, /*   .XXXXX..   */
+	/* 234 = 0xea 90*/
+	0x00, /*   ........   */
+	0x72, /*   .XXX..X.   */
+	0xfe, /*   XXXXXXX.   */
+	0x8e, /*   X...XXX.   */
+	0x80, /*   X.......   */
+	0x8e, /*   X...XXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x72, /*   .XXX..X.   */
+	/* 235 = 0xeb 90*/
+	0x00, /*   ........   */
+	0x8c, /*   X...XX..   */
+	0x9e, /*   X..XXXX.   */
+	0xb2, /*   X.XX..X.   */
+	0xf2, /*   XXXX..X.   */
+	0x5e, /*   .X.XXXX.   */
+	0x0c, /*   ....XX..   */
+	0x00, /*   ........   */
+	/* 236 = 0xec 90*/
+	0x18, /*   ...XX...   */
+	0x3c, /*   ..XXXX..   */
+	0x24, /*   ..X..X..   */
+	0x3c, /*   ..XXXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x24, /*   ..X..X..   */
+	0x3c, /*   ..XXXX..   */
+	0x18, /*   ...XX...   */
+	/* 237 = 0xed 90*/
+	0x98, /*   X..XX...   */
+	0xfc, /*   XXXXXX..   */
+	0x64, /*   .XX..X..   */
+	0x34, /*   ..XX.X..   */
+	0x2c, /*   ..X.XX..   */
+	0x3e, /*   ..XXXXX.   */
+	0x1b, /*   ...XX.XX   */
+	0x01, /*   .......X   */
+	/* 238 = 0xee 90*/
+	0x00, /*   ........   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0x92, /*   X..X..X.   */
+	0xd6, /*   XX.X.XX.   */
+	0x7c, /*   .XXXXX..   */
+	0x38, /*   ..XXX...   */
+	0x00, /*   ........   */
+	/* 239 = 0xef 90*/
+	0x00, /*   ........   */
+	0x7e, /*   .XXXXXX.   */
+	0xfe, /*   XXXXXXX.   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0xfe, /*   XXXXXXX.   */
+	0x7e, /*   .XXXXXX.   */
+	/* 240 = 0xf0 90*/
+	0x00, /*   ........   */
+	0x54, /*   .X.X.X..   */
+	0x54, /*   .X.X.X..   */
+	0x54, /*   .X.X.X..   */
+	0x54, /*   .X.X.X..   */
+	0x54, /*   .X.X.X..   */
+	0x54, /*   .X.X.X..   */
+	0x00, /*   ........   */
+	/* 241 = 0xf1 90*/
+	0x00, /*   ........   */
+	0x22, /*   ..X...X.   */
+	0x22, /*   ..X...X.   */
+	0xfa, /*   XXXXX.X.   */
+	0xfa, /*   XXXXX.X.   */
+	0x22, /*   ..X...X.   */
+	0x22, /*   ..X...X.   */
+	0x00, /*   ........   */
+	/* 242 = 0xf2 90*/
+	0x00, /*   ........   */
+	0x02, /*   ......X.   */
+	0x22, /*   ..X...X.   */
+	0x72, /*   .XXX..X.   */
+	0xda, /*   XX.XX.X.   */
+	0x8a, /*   X...X.X.   */
+	0x02, /*   ......X.   */
+	0x00, /*   ........   */
+	/* 243 = 0xf3 90*/
+	0x00, /*   ........   */
+	0x02, /*   ......X.   */
+	0x8a, /*   X...X.X.   */
+	0xda, /*   XX.XX.X.   */
+	0x72, /*   .XXX..X.   */
+	0x22, /*   ..X...X.   */
+	0x02, /*   ......X.   */
+	0x00, /*   ........   */
+	/* 244 = 0xf4 90*/
+	0x60, /*   .XX.....   */
+	0xe0, /*   XXX.....   */
+	0x80, /*   X.......   */
+	0xff, /*   XXXXXXXX   */
+	0x7f, /*   .XXXXXXX   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 245 = 0xf5 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0xfe, /*   XXXXXXX.   */
+	0xff, /*   XXXXXXXX   */
+	0x01, /*   .......X   */
+	0x07, /*   .....XXX   */
+	0x06, /*   .....XX.   */
+	/* 246 = 0xf6 90*/
+	0x00, /*   ........   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0xd6, /*   XX.X.XX.   */
+	0xd6, /*   XX.X.XX.   */
+	0x10, /*   ...X....   */
+	0x10, /*   ...X....   */
+	0x00, /*   ........   */
+	/* 247 = 0xf7 90*/
+	0x00, /*   ........   */
+	0x48, /*   .X..X...   */
+	0x6c, /*   .XX.XX..   */
+	0x24, /*   ..X..X..   */
+	0x6c, /*   .XX.XX..   */
+	0x48, /*   .X..X...   */
+	0x6c, /*   .XX.XX..   */
+	0x24, /*   ..X..X..   */
+	/* 248 = 0xf8 90*/
+	0x00, /*   ........   */
+	0x60, /*   .XX.....   */
+	0xf0, /*   XXXX....   */
+	0x90, /*   X..X....   */
+	0x90, /*   X..X....   */
+	0xf0, /*   XXXX....   */
+	0x60, /*   .XX.....   */
+	0x00, /*   ........   */
+	/* 249 = 0xf9 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x18, /*   ...XX...   */
+	0x18, /*   ...XX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 250 = 0xfa 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x08, /*   ....X...   */
+	0x08, /*   ....X...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 251 = 0xfb 90*/
+	0x00, /*   ........   */
+	0x80, /*   X.......   */
+	0x80, /*   X.......   */
+	0xff, /*   XXXXXXXX   */
+	0xff, /*   XXXXXXXX   */
+	0x03, /*   ......XX   */
+	0x06, /*   .....XX.   */
+	0x04, /*   .....X..   */
+	/* 252 = 0xfc 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x78, /*   .XXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x80, /*   X.......   */
+	0xf8, /*   XXXXX...   */
+	0xf8, /*   XXXXX...   */
+	0x00, /*   ........   */
+	/* 253 = 0xfd 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x48, /*   .X..X...   */
+	0xe8, /*   XXX.X...   */
+	0xb8, /*   X.XXX...   */
+	0x98, /*   X..XX...   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 254 = 0xfe 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x3c, /*   ..XXXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x3c, /*   ..XXXX..   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	/* 255 = 0xff 90*/
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+	0x00, /*   ........   */
+};
+#endif
